src/ussd_popi_rapidpro.js

```diff
@@ -975,8 +1132,10 @@ go.app = function() {
                 next: function(choice) {
                     if(_.includes(["miscarriage", "stillbirth", "babyloss"], choice.value)) {
                         return "state_loss_messages";
+                    } else if(_.includes(["not_useful", "unknown"], choice.value)) {
+                        return "state_message_unhelpful_or_unknown";
```

Comment:
I think this should still go to state_submit_opt_out for these reasons. maybe split after that to show the specific messages.

Reply:
this comment is not fixed yet?